nucypher/control/controllers.py

```diff
@@ -312,6 +312,11 @@ def handle_request(self, method_name, control_request, *args, **kwargs) -> Respo
             request_body = control_request.data or dict()
             if request_body:
                 request_body = json.loads(request_body)
+
+            # handle query string parameters for GET methods
+            if control_request.method == 'GET' and hasattr(control_request, 'args'):
```

Comment:
@vepkenez just pushed a commit, wdyt about adding this additional check?

Reply:
POST/PATCH/WHATEVER requests can also have query params.
I say lets honor them and not check the request method.  The question is... which should take precedence?

Should they supersede the request_body?
Should we raise a 400 if the same key appears in both?

It's kind of an edge case and not worth spending a lot of time on in my opinion.

Reply:
Ok. I'll remove the check.

Come to think of it, I'll also add something for the docs before merging.

Reply:
✔️